src/microbe_stage/MicrobeAI.cs

```diff
@@ -305,8 +305,15 @@ private void PursueAndConsumeChunks(Vector3 chunk, Random random)
             MoveWithRandomTurn(0.1f, 0.2f, random);
         }
 
-        // Always set target Position, for use later in AI
-        microbe.MovementDirection = new Vector3(0.0f, 0.0f, -Constants.AI_BASE_MOVEMENT);
+        // If this Microbe is right on top of the chunk, stop instead of spinning
+        if (DistanceFromMe(chunk) < 1.0f)
```

Comment:
Instead of an arbitrary number would this be better if this compares against the microbe's `Radius` property? I think that would lead to a behavior where cells would stop immediately once a chunk enters its membrane area.

Reply:
I'll play around with it, but I suspect that that would have the same problem that caused this bug in the first place: radius pretends that the microbe is a circle, but absorption is based on shape collision.

Reply:
ok, it worked.

Reply:
Well the radius is the *maximum* distance the membrane is from the microbe center. I guess it is possible to make a shape that this logic would break for. Perhaps the AI could slow down but still approach the chunk location?

Reply:
I would expect any non-zero speed to result in the spinning behavior (although maybe we could slow it down). I would also expect issues with very slow cells now looking like idiots slowing to a crawl right before grabbing a chunk.

I've tested using radious with a microbe that's just a horizontal line and it does work, which leads me to think that radious might be what's used to determine engulfing. Setting it back to 1.0f would be safest, but would result in that one-time 180 degree turn every time a microbe eats a chunk.

Reply:
I'm still wondering why that happens. Only if the chunk doesn't actually exist can there be a coordinate an AI cell wants to orbit. If the chunk actually exists, the cell would push it, causing the target position to also drift away...
I don't think using Radius here is good because otherwise the players can find a way to cheese the AI by making a really wide cell that stops before reaching a chunk.